elements/lisk-p2p/src/peer_book/peer_book.ts

```diff
@@ -149,4 +151,23 @@ export class PeerBook {
 
 		return false;
 	}
+
+	public getDiscoveryPeerList(
+		minPeerListLength: number,
```

Comment:
Can you keep the previous names for the variables? Also the name of this function is not very straightforward to understand.